docs/framework/configure-apps/file-schema/runtime/generatepublisherevidence-element.md

```diff
@@ -49,7 +49,7 @@ Specifies whether the runtime creates <xref:System.Security.Policy.Publisher> ev
 ## Remarks  
   
 > [!NOTE]
-> In the .NET Framework 4 and later, this element has no effect on assembly load times. For more information, see the "Security Policy Simplification" section in [Security Changes](../../../security/security-changes.md).  
+> In the .NET Framework 4 and later, this element has no effect on assembly load times. For more information, see the "Security Policy Simplification" section in [Security Changes](https://docs.microsoft.com/previous-versions/dotnet/framework/security/security-changes).  
```

Comment:
I think we can use the root without domain.

```suggestion
> In the .NET Framework 4 and later, this element has no effect on assembly load times. For more information, see the "Security Policy Simplification" section in [Security Changes](/previous-versions/dotnet/framework/security/security-changes).  
```

Reply:
I think the only reason to drop the domain is for the links to work in offline books, and /previous-versions/ docs aren't part of offline books. Also, /azure/ docs aren't in offline books. Most of the rest are though (windows, sql, aspnet, etc.).